### PR TITLE
add "fixture flowchart" to docs

### PR DESCRIPTION
**Pull request by boneskull**
- also add some references to global fixtures

@munter You know more about web stuff than I do.  Is this a reasonable way to use an SVG?  In particular, I wanted to make portions of it _clickable_, and couldn't figure out how to do that without embedding the `<svg>` into the page.

### CODE REVIEW

File: docs/_includes/fixture-wizard.html [hunk 1]
```diff
@@ -0,0 +1,419 @@
+<svg width="854px" height="665px" viewBox="0 0 854 665" version="1.1">
```
Comment (craigtaub) [inline docs/_includes/fixture-wizard.html:1]: Fixed width, think will break widths on mobile.

Comment (boneskull) [reply docs/_includes/fixture-wizard.html:1]: ah hmm. I don't know SVG very well but I'll see if I can it responsive.

File: docs/_includes/fixture-wizard.html [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+          font-size="16"
+          font-weight="normal"
+          fill="#000000"
+        >
+          <tspan x="17.144" y="24">NO</tspan>
+        </text>
+      </g>
+    </g>
+  </g>
+</svg>
```
Comment (outsideris) [inline docs/_includes/fixture-wizard.html:419]: It looks good. 
What tools do you use for making this diagram? Writing a guide as a comment for how we can modify this diagram is better for later.

Comment (boneskull) [reply docs/_includes/fixture-wizard.html:419]: I used Sketch for this.  I can add the source `.sketch` file to the repo

**Comment by boneskull** [comment]
OK, I think I've addressed the responsiveness issue. I've also added a `assets/mocha-fixture-wizard.sketch` file to the repo

File: docs/index.md [hunk 3]
```diff
@@ -1653,6 +1659,13 @@ describe('my API', function() {
 
 Finally, use this command to bring it together: `mocha --require fixtures.mjs test.spec.mjs`.
 
+## Test Fixture Decision-Tree Wizard Thing
+
+This flowchart will help you decide which of [hooks], [root hook plugins] or
+[global fixtures] you should use.
+
+{% include fixture-wizard.html %}
```
Comment (outsideris) [inline docs/index.md:1667]: Including HTML is not handled properly and [SVG is broken](https://deploy-preview-4440--mocha.netlify.app/#test-fixture-decision-tree-wizard-thing). 
I don't know why even I checked it.

**Review by outsideris — CHANGES_REQUESTED** [review]
Including SVG is broken.

File: docs/index.md (hunk 3 above)
Comment (boneskull) [reply docs/index.md:1667]: ohh I know what happened.  prettier broke it.  I am not sure if this is a bug in prettier or liquid or eleventy, but when prettier reformats it, the svg breaks.

Comment (boneskull) [reply docs/index.md:1667]: OK, wrapping the svg in a div works.  I think this is an 11ty bug

Comment (boneskull) [reply docs/index.md:1667]: see https://github.com/11ty/eleventy/issues/1442

**Review by outsideris — APPROVED** [review]

**Comment by boneskull** [comment]
(semver minor b/c it references other stuff that will land in a minor)

**Pull request merged**